core/src/test/java/hudson/scheduler/CronTabTest.java

```diff
@@ -340,6 +341,24 @@ public int next(int n) {
         assertEquals("[35, 56]", times.toString());
     }
 
+    @Test public void floorCeilMinuteGranularity() throws Exception {
+        var tab = new CronTab("*/5 * * * *");
+        assertFloorCeil(tab, new GregorianCalendar(2025, 2, 4, 14, 15, 23), "Mar 4, 2025, 2:15:00 PM", "Mar 4, 2025, 2:20:00 PM");
+        assertFloorCeil(tab, new GregorianCalendar(2025, 2, 4, 14, 19, 23), "Mar 4, 2025, 2:15:00 PM", "Mar 4, 2025, 2:20:00 PM");
+        assertFloorCeil(tab, new GregorianCalendar(2025, 2, 4, 14, 16,  0), "Mar 4, 2025, 2:15:00 PM", "Mar 4, 2025, 2:20:00 PM");
+        assertFloorCeil(tab, new GregorianCalendar(2025, 2, 4, 14, 19,  0), "Mar 4, 2025, 2:15:00 PM", "Mar 4, 2025, 2:20:00 PM");
+        assertFloorCeil(tab, new GregorianCalendar(2025, 2, 4, 14, 15,  0), "Mar 4, 2025, 2:15:00 PM", "Mar 4, 2025, 2:15:00 PM");
+    }
+
+    private static void assertFloorCeil(CronTab tab, Calendar now, String expectedFloor, String expectedCeil) {
+        var fmt = DateFormat.getDateTimeInstance(DateFormat.MEDIUM, DateFormat.MEDIUM);
```

Comment:
Fails on my machine due to my French locale.

```suggestion
        var fmt = DateFormat.getDateTimeInstance(DateFormat.MEDIUM, DateFormat.MEDIUM, Locale.ENGLISH);
```

Reply:
Right, U+202F NARROW NO-BREAK SPACE used in some Java versions.